aspnetcore-webapi/RateLimitingDotNET8/RateLimitingDotNET8/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.RateLimiting;
+using System.Diagnostics;
+
+namespace RateLimitingDotNET8.Controllers;
+
+[EnableRateLimiting("fixed")]
```

Comment:
Instead of hardcoded string, using constant Policies.Fixed as we used in Program.cs